src/pages/iou/WaypointEditor.tsx

```diff
@@ -103,15 +85,15 @@ function WaypointEditor({route: {params: {iouType = '', transactionID = '', wayp
         }
     }, [parsedWaypointIndex, waypointCount]);
 
-    const waypointAddress = lodashGet(currentWaypoint, 'address', '');
-    const isEditingWaypoint = Boolean(threadReportID);
+    const waypointAddress = currentWaypoint.address ?? '';
+    const isEditingWaypoint = !!threadReportID;
     // Hide the menu when there is only start and finish waypoint
     const shouldShowThreeDotsButton = waypointCount > 2;
     const shouldDisableEditor =
         isFocused &&
         (Number.isNaN(parsedWaypointIndex) || parsedWaypointIndex < 0 || parsedWaypointIndex > waypointCount || (filledWaypointCount < 2 && parsedWaypointIndex >= waypointCount));
 
-    const validate = (values) => {
+    const validate = (values: Record<string, string>) => {
```

Comment:
Looking at `FormProvider`, it seems that `validate` prop passes the result of `ValidationUtils.prepareValues()`, so the type would probably be `ValuesType` from `src/libs/ValidationUtils.ts`.

Reply:
yeah , but `ValuesType` is a `Record<string, unknown>` but in that particular case I checked and we are getting `Record<string, string>` so I think its more accurate , wdyt @fabioh8010 ?